src/modem/begin-command.cpp

```diff
@@ -0,0 +1,50 @@
+#include "modem/commands.h"
```

Comment:
I have no strong preference, personally I use -s for the file name and _s for file metadata. like file-name_test.cc or file-name_main.cc. But if we want to standardize on something else I am ok with that too.